src/components/header/Search.tsx

```diff
@@ -71,6 +74,17 @@ export function Search() {
         }}
         ref={SearchInputRef}
       />
+      {searchTerm && (
+        <div
+          role='status'
+          className={clsx('epr-status-search-results', 'epr-status-visually-hidden')}
+          aria-live='polite'
+          id='epr-search-id'
+          aria-atomic='true'
+        >
+          {searchResultsCount > 0 ? statusSearchResults : statusNoSearchResults}
+        </div>
+      )}
```

Comment:
I try to avoid the conditional shorthand, and be more explicit when conditionally rendering elements.
```suggestion
      {searchTerm ? (
        <div
          role='status'
          className={clsx('epr-status-search-results', 'epr-status-visually-hidden')}
          aria-live='polite'
          id='epr-search-id'
          aria-atomic='true'
        >
          {searchResultsCount > 0 ? statusSearchResults : statusNoSearchResults}
        </div>
      ) : null}
```

If it looks to verbose, you can move it to a new component.

Reply:
Updated!